tools/python/convert_from_tensorflow.py

```diff
@@ -371,6 +376,68 @@ def dump_mathunary_to_file(self, node, f):
         output_operand_index = self.add_operand(node.name, Operand.IOTYPE_OUTPUT)
         np.array([output_operand_index],dtype=np.uint32).tofile(f)
 
+    def dump_batchnormalization_to_file(self, node, f):
+        node_name_list = []
+        node_name_list = node.name.split('/')
+        if node.name in self.output_names:
+            node_name_list.append('add_1')
+        name_dict = {node_name_list[1]:node.op}
+        self.batchnorm_name_dict.update(name_dict)
+        self.converted_nodes.add(node.name)
+        if node_name_list[1] == 'add':
+            i_variance = self.name_node_dict[node.input[0]]
+            self.batchnorm_node.update({'i_variance':i_variance})
+            i_variance_eps = self.name_node_dict[node.input[1]]
+            self.batchnorm_node.update({'i_variance_eps':i_variance_eps})
+        elif node_name_list[1] == 'Rsqrt':
+            i_rsqrt = self.name_node_dict[node.input[0]]
+            self.batchnorm_node.update({'Rsqrt':i_rsqrt})
+        elif node_name_list[1] == 'mul':
+            i_scale = self.name_node_dict[node.input[1]]
+            self.batchnorm_node.update({'i_scale':i_scale})
+        elif node_name_list[1] == 'mul_1':
+            i_bn_input = self.name_node_dict[node.input[0]]
+            self.batchnorm_node.update({'i_bn_input':i_bn_input})
+        elif node_name_list[1] == 'mul_2':
+            i_mean = self.name_node_dict[node.input[0]]
+            self.batchnorm_node.update({'i_mean':i_mean})
+        elif node_name_list[1] == 'sub':
+            i_offset = self.name_node_dict[node.input[0]]
+            self.batchnorm_node.update({'i_offset':i_offset})
+        elif node_name_list[1] == 'add_1':
+            o_bn_output = node
+            self.batchnorm_node.update({'batchnorm_output':o_bn_output})
```

Comment:
maybe we can change these into something like
`if node.name in self.verification_batchnorm_name_dict`
instead of these if clause?